cli/scan/scan.go

```diff
@@ -35,17 +43,29 @@ func scanMain(args []string, c cli.Config) (err error) {
 			if err != nil {
 				return
 			}
+			hosts++
 
-			fmt.Printf("Scanning %s...\n", host)
```

Comment:
I think it's still worth printing out the hosts being scanned, even they're just all printed at once.